posts/aws-appsync-is-coming-for-your-api.md

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
You should add some context around your example application sooner that where it is right now. Readers might be surprised when reading "unicorn", "carnival", "vote" and such. Give 'em context!

Reply:
I've modified the text slightly to mention the app.